Cargo.toml

```diff
@@ -3,28 +3,30 @@ name = "rustodon"
 version = "0.1.0"
 authors = ["The Rustodon team"]
 
+[workspace]
+members = ['database/', 'lib/resopt/']
+
 [dependencies]
 chrono = "0.4"
 dotenv = "0.10"
 failure = "0.1.1"
 failure_derive = "0.1.1"
 lazy_static = "1.0"
 itertools = "0.7.4"
-
+resopt = { path = "lib/resopt/" }
 rocket = { git = "https://github.com/SergioBenitez/Rocket", branch = "v0.3" }
 rocket_codegen = { git = "https://github.com/SergioBenitez/Rocket", branch = "v0.3" }
 
 diesel = { version = "1.0.0", features = ["postgres", "chrono"] }
```

Comment:
I think `diesel` and all related items (`r2d2-diesel`, `diesel_infer_schema`, and probably also `r2d2` since I think nothing else is using connection pooling right now) can be removed from the `Cargo.toml` for the rustodon crate.  This'll also involve removing `extern crate` lines from `src/main.rs` (keeping them in `database/src/lib.rs` for proper linkage there, of course).

(I found this to be helpful while porting my WIP sqlite changes to the new crate structure, because it meant I didn't have to infer features in two places)